### DISCUSSION TITLE
PCM60X?

### DISCUSSION THREAD

**Discussion by Seedser** [jblance/mpp-solar#104] [General]
Will this work with the MPP PCM60x?

**Comment by jblance** [comment]
It depends on the protocol used on that inverter and how you connect to it.
In short, it should and if it doesn't it probably isn't hard to make it work.
How do you connect?
Do you have a Linux type system to connect with?

**Reply by Seedser** [reply]
I haven't connected to PCM60X other than the MPP solar windows app.  I am looking for option to get data out to my MQTT broker.  I would run in a RPI3 or 4.  I would have option for laptop running Ubuntu.

**Reply by jblance** [reply]
Then I'm pretty sure it will either work ok or I can make it work. If you want more certainty you could do a wireshark trace of the windows program communications to the inverter and I can compare to the currently implemented protocols
